packages/state-transition/src/util/weakSubjectivity.ts

```diff
@@ -111,7 +111,7 @@ export function isWithinWeakSubjectivityPeriod(
   wsState: BeaconStateAllForks,
   wsCheckpoint: Checkpoint
 ): boolean {
-  const wsStateEpoch = computeEpochAtSlot(wsState.slot);
+  const wsStateEpoch = Math.ceil(wsState.slot / SLOTS_PER_EPOCH);
```

Comment:
@g11tech do we really need this for the fix? it's quite confusing here so either not to include this change, or drop some comments to clarify